src/plots/stackedBarPlot.ts

```diff
@@ -151,7 +151,7 @@ export class StackedBar<X, Y> extends Bar<X, Y> {
       if (maximum.extent !== baselineValue) {
         // only draw sums for values not at the baseline
 
-        const text = this.labelFormatter()(maximum.extent);
+        const text = this.labelFormatter()(maximum.extent, undefined, undefined, undefined);
```

Comment:
Yeah it's gross. A single stack is built from an array of datums, each one coming from a different dataset. How do we reconcile the two?